target/linux/ipq40xx/patches-5.15/903-add-a1300-dts-file.patch

```diff
@@ -0,0 +1,10 @@
+--- a/arch/arm/boot/dts/Makefile
++++ b/arch/arm/boot/dts/Makefile
+@@ -951,6 +951,7 @@ dtb-$(CONFIG_ARCH_QCOM) += \
+ 	qcom-apq8084-mtp.dtb \
+ 	qcom-ipq4018-ap120c-ac.dtb \
+ 	qcom-ipq4018-ap120c-ac-bit.dtb \
++	qcom-ipq4018-gl-a1300.dtb \
```

Comment:
See https://github.com/openwrt/openwrt/commit/4d8b42d8a7774070ac0439915f8de1430db9a8e3, we don't need to add this file now.